style.css

```diff
@@ -3,6 +3,491 @@
 html {
   box-sizing: border-box;
   font-size: 16px;
-  font-family: 'Inter', sans-serif;
+  font-family: 'Lato', sans-serif;
   scroll-behavior: smooth;
 }
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+:root {
+  --color-primary: #fff;
+  --color-secondary: #f7f7f8;
+  --color-tertiary: #272a31;
+  --color-quaternary: #d3d3d3;
+  --color-quinary: #ff4f38;
+}
+
+body {
+  background-color: var(--color-primary);
+  margin: 0 auto;
+}
+
```

Comment:
- [ ] **Good design so far 👏 👏** However, It would be better to keep a consistent margin of your webpage as given on the original design, this will make your page more professional 👍 👍 

![image](https://github.com/RuriEnciso/capstone1-concert-web/assets/102757126/40812493-1f06-4a8f-a25e-ca43e75bd7e2)